docs/runtime_api.md

```diff
@@ -600,8 +600,6 @@ As such, `PlainMessage<T>` can be a great fit to use in throughout your business
 if that business logic is never concerned with serialization, and does not need 
 `instanceof`.
 
-Note that any `T` is assignable to `PlainMessage<T>`.
```

Comment:
Rather than delete this, let's maybe say:

> Note that any T that extends `Message` is assignable to `PlainMessage<T>`

Reply:
Maybe we just clarify based on what you said above:

> Note that any `T` (assuming `T` extends `Message`) is assignable to a variable of type `PlainMessage<T>`.

Reply:
I like that.